t/unit/bin/test_worker.py

```diff
@@ -8,6 +8,7 @@
 from billiard.process import current_process
 from case import Mock, mock, patch, skip
 from kombu import Exchange, Queue
+from six.moves import reload_module
```

Comment:
isn't it available in the five module?

Reply:
Right. Updated.

Reply:
Looks like five isn't there in 2.7...

Reply:
No. Use five.
I use it later to find code to remove for Celery 5.